bindings/nodejs/lib/wallet/WalletMethodHandler.ts

```diff
@@ -71,6 +75,19 @@ export class WalletMethodHandler {
     }
 
     async getClient(): Promise<Client> {
-        return new Client(await getClientFromWallet(this.methodHandler));
+        return new Promise((resolve, reject) => {
+            getClientFromWallet(this.methodHandler).then((result: any) => {
+                if (result.message !== undefined) {
+                    if (result.message !== undefined) {
```

Comment:
ugh, what?

Reply:
That happens when you quickly copy paste and it works fine so you don't look at it anymore :trollface: